docs/electron.rst

```diff
@@ -242,3 +242,21 @@ on ``$PATH``:
       # Install app wrapper
       - install -Dm755 -t /app/bin/ ../run.sh
 
+Settings in package.json
+------------------------
```

Comment:
I'd say that electron-builder settings in `package.json` isn't worth its own paragraph. Maybe make it a sub-section of "Build commands", or add a new "Tips and tricks section"?

Reply:
Done